js/challenges/001_EvenNumbers/index.mjs

```diff
@@ -0,0 +1,11 @@
+/* 
+ * Returns a new array with only the odd numbers from the original, if any.
+ * @param {array} arr Original Array
+ * @returns {array} New odd-only array
+ */
```

Comment:
I can't begin to explain how happy it made me to see you properly header your method, 🥲 so happy